src/app/settings/api-info/api-info.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { BACKEND_API_URL, ConfigService, NotificationService } from '../../shared/services';
+import { UserService } from '../../shared/services/user.service';
+import { Observable } from 'rxjs/Observable';
+
+
+interface ApiInfoTextField {
+  title: string;
+  value: string;
+}
+
+interface ApiInfoLink {
+  title: string;
+  href: string;
+}
+
+@Component({
+  selector: 'cs-api-info',
+  templateUrl: 'api-info.component.html',
+  styleUrls: ['api-info.component.scss']
+})
+export class ApiInfoComponent implements OnInit {
+  public linkFields: Array<ApiInfoLink>;
+  public inputFields: Array<ApiInfoTextField>;
+
+  constructor(
+    private configService: ConfigService,
+    private notificationService: NotificationService,
+    private userService: UserService
+  ) {}
+
+  public ngOnInit(): void {
+    Observable.forkJoin(
+      this.userService.getApiKey(),
+      this.userService.getSecretKey(),
+      this.configService.get('API_DOC_LINK')
+    )
+      .subscribe(([apiKey, secretKey, apiDocLink]) => {
+        this.linkFields = [
+          { title: 'API_URL', href: this.apiUrl },
+          { title: 'API_DOC_LINK', href: apiDocLink }
+        ];
+
+        this.inputFields = [
+          { title: 'API_KEY', value: apiKey },
+          { title: 'API_SECRET_KEY', value: secretKey }
+        ];
+      });
+  }
+
+  private get apiUrl(): string {
+    return location.origin + BACKEND_API_URL;
```

Comment:
Do all the browsers support localtion.origin?
may be we need to add this?
` if (!window.location.origin) {
        window.location.origin = '' +
            window.location.protocol + '//' +
            window.location.hostname +
            (window.location.port ? ':' + window.location.port : '');
    }`